### PR TITLE
Prefix for old flexbox implementations

### LINKED ISSUE #156: Support prefixing flexbox
Now that vendor prefixing has landed it would be nice to support flexbox. On Safari and IE, the display property needs to be prefixed.

Here's an example implementation:
https://gist.github.com/wmertens/78c840ce04b6e9a1193d#file-auto-vendor-prefixer-coffee-L37-L59

I'm not up to implementing this myself in Radium at the moment, so opening this issue instead.

Cc @ianobermiller

### PR DESCRIPTION
**Pull request by ianobermiller**
Fixes #156

### CODE REVIEW

**Comment by ianobermiller** [comment]
Can someone with access to IE10 test this?

**Comment by alexlande** [comment]
I can test it this evening or tomorrow morning. http://dev.modern.ie/tools/vms/ if anyone has time/interest before then.

**Comment by ianobermiller** [comment]
I did a simple test on IE10, and it works: http://jsbin.com/dogosivano/2/edit?html,js,console,output

File: modules/prefix.js [hunk 1]
```diff
@@ -148,6 +197,11 @@ var _getPrefixedValue = function (property, value) {
 // and values.
 var prefix = function (style, mode /* 'css' or 'js' */) {
   mode = mode || 'js';
+
+  if (!ExecutionEnvironment.canUseDOM) {
+    return style;
```
Comment (wmertens) [inline modules/prefix.js]: Minor nitpick: The executionenvironment will never change, so why not check at module load time and make prefix the unity function if not canUseDOM?

**Comment by wmertens** [comment]
:beers:

**Pull request merged**